<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.1.1 (Python 3.11.5 on darwin)" generated="2024-12-05T02:59:43.409553" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="/Users/aappo/.vscode/extensions/robocorp.robotframework-lsp-1.13.0/src/robotframework_ls/vendored/robotframework_interactive/robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2024-12-05T02:59:43.418647" elapsed="12.329080"/>
</kw>
<status status="PASS" start="2024-12-05T02:59:43.418364" elapsed="12.329721"/>
</test>
<status status="PASS" start="2024-12-05T02:59:43.410154" elapsed="12.338522"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2024-12-05T02:59:43.407106" level="WARN">Error in file '/Users/aappo/.vscode/extensions/robocorp.robotframework-lsp-1.13.0/src/robotframework_ls/vendored/robotframework_interactive/robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
